controllers/applicationvolumereplication_controller.go

```diff
@@ -195,658 +276,895 @@ func (r *ApplicationVolumeReplicationReconciler) Reconcile(ctx context.Context,
-	if err != nil {
-		a.log.Info("Failed to validate new home cluster selection for subscription",
-			"name", subscription.Name, "newHomeCluster", newHomeCluster, "errMsg", err.Error())
+	if a.needStatusUpdate {
+		if err := a.updateAVRStatus(); err != nil {
+			a.log.Error(err, "failed to update status")
 
-		// UNPAUSE if and only if the current cluster is the newly selected cluster. This inidicates
-		// that the user has selected to failover/failback to the same current cluster.
-		return unpause && (errorswrapper.Is(err, ErrSameHomeCluster))
+			return requeue
+		}
 	}
 
-	a.log.Info("Found new home cluster", "name", newHomeCluster)
+	requeue = !done
+	a.log.Info("Completed processing placement", "requeue", requeue)
 
-	mwName := BuildManifestWorkName(subscription.Name, subscription.Namespace, MWTypePV)
+	return requeue
+}
 
-	pvMW, err := a.findManifestWork(mwName, newHomeCluster)
-	if err != nil {
-		a.log.Error(err,
-			fmt.Sprintf("Failed to find 'PV restore' ManifestWork for subscription %s", subscription.Name))
-		// DON'T unpause
-		return !unpause
-	}
+func (a *AVRInstance) processPlacement() (bool, error) {
+	a.log.Info("Process AVR Placement", "name", a.avrPlacementRule)
 
-	if pvMW != nil {
-		a.log.Info(fmt.Sprintf("Found manifest work (%v)", pvMW))
-		// Unpause if the ManifestWork has been applied
-		return IsManifestInAppliedState(pvMW)
+	switch a.instance.Spec.Action {
+	case rmn.ActionFailover:
+		return a.runFailover()
+	case rmn.ActionFailback:
+		return a.runFailback()
+	case rmn.ActionRelocate:
+		return a.runRelocate()
 	}
 
-	return a.cleanupAndRestore(subscription, newHomeCluster)
+	// Not a failover, a failback, or a relocation.  Must be an initial deployment.
+	return a.runInitialDeployment()
 }
 
-func (a *AVRInstance) cleanupAndRestore(
-	subscription *subv1.Subscription,
-	newHomeCluster string) bool {
-	const unpause = true
+func (a *AVRInstance) runInitialDeployment() (bool, error) {
+	a.log.Info("Running initial deployment")
+	a.resetDRState()
 
-	pvMWName := fmt.Sprintf(ManifestWorkNameFormat, subscription.Name, subscription.Namespace, MWTypePV)
+	const done = true
 
-	err := a.deleteExistingManifestWork(subscription, pvMWName)
-	if err != nil {
-		a.log.Error(err, "Failed to delete existing PV manifestwork for subscription", "name", subscription.Name)
+	// 1. Check if the user wants to use the preferredCluster
+	homeCluster := ""
+	homeClusterNamespace := ""
 
-		return !unpause
+	if a.instance.Spec.PreferredCluster != "" {
+		homeCluster = a.instance.Spec.PreferredCluster
+		homeClusterNamespace = homeCluster
 	}
 
-	vrgMWName := fmt.Sprintf(ManifestWorkNameFormat, subscription.Name, subscription.Namespace, MWTypeVRG)
-
-	err = a.deleteExistingManifestWork(subscription, vrgMWName)
-	if err != nil {
-		a.log.Error(err, "Failed to delete existing VRG manifestwork for subscription", "name", subscription.Name)
+	if homeCluster == "" && len(a.avrPlacementRule.Status.Decisions) != 0 {
```

Comment:
A dummy reconcile for an `nil` action in AVR, may result in switching clusters if clonePlRule is updated.

Assume initial placement was complete:
1. current cluster went offline and recovered, and clonePlRule changed its decision in the time period
2. A reconcile for AVR is triggered (even as a controller restart say, IOW no spec change)
3. The workload would be placed in the new cluster, without any failover/back actions, rather than being left where it is

Can possibly be rectified for dynamic initial placement, if we decide to check the user PlRule decision exists and just leave it there (or check status.preferredCluster value and not change the same) .